xdsl/frontend/onnx/ir_builder.py

```diff
@@ -0,0 +1,25 @@
+from onnx import ValueInfoProto
+
+from xdsl.frontend.onnx.type import get_type
+from xdsl.ir import Attribute
+
+
+class OnnxXdslMapping:
+    """The representation of the onnx context."""
+
+    type_by_name: dict[str, Attribute]
+
+    def __init__(self):
+        self.type_by_name = {}
+
+
+def visit_value_info(i: ValueInfoProto, ctx: OnnxXdslMapping) -> Attribute:
+    """Given the onnx ValueInforProto, it returns the corresponding Attribute stored in the context."""
```

Comment:
```suggestion
    """Given the onnx ValueInfoProto, it returns the corresponding Attribute stored in the context."""
```

typo but also a few more words on this? is this a visitor pattern? why? what is the point of taking a `ctx`? etc